data_utils/helper_functions.py

```diff
@@ -33,7 +36,9 @@ def order_candidates(candidate_neighbours: dict, distance_matrix: [[float]]):
         Matrix containing the distances for each pair of atoms within the structure.
     """
 
-    print("Ordering candidate neighbours based on their distance.")
+    print_distributed(
+        verbosity, "Ordering candidate neighbours based on their distance."
```

Comment:
We will probably want to remove prints like this now that we'll usually have to create edges for each graph in the dataset

Reply:
I just replaced existing prints with ones that are practical for distributed computing. If you don't need the print, and this will be addressed by another PR, then you will remove this print as you would have done with the standard one.